UWP build broken with nightly update

```
warning: Patch `rustc-std-workspace-std v1.99.0 (C:\Users\manis\.rustup\toolchains\nightly-2019-11-16-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\tools\rustc-std-workspace-std)` was not used in the crate graph.

...

   Compiling rustc-std-workspace-alloc v1.99.0 (C:\Users\manis\.rustup\toolchains\nightly-2019-11-16-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\tools\rustc-std-workspace-alloc)
error[E0635]: unknown feature `compiler_builtins_lib`
   --> C:\Users\manis\.rustup\toolchains\nightly-2019-11-16-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\libstd\lib.rs:247:12
    |
247 | #![feature(compiler_builtins_lib)]
    |            ^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error
```

<details>

```
C:\Users\manis\dev\servo>python mach build --uwp --win-arm64 -r
info: component 'rust-src' is up to date
    Updating crates.io index
warning: Patch `rustc-std-workspace-std v1.99.0 (C:\Users\manis\.rustup\toolchains\nightly-2019-11-16-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\tools\rustc-std-workspace-std)` was not used in the crate graph.
Check that the patched package version and available features are compatible
with the dependency requirements. If the patch has a different version from
what is locked in the Cargo.lock file, run `cargo update` to use the new
version. This may also occur with an optional dependency that is not enabled.
   Compiling compiler_builtins v0.1.21
   Compiling core v0.0.0 (C:\Users\manis\.rustup\toolchains\nightly-2019-11-16-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\libcore)
   Compiling cc v1.0.47
   Compiling libc v0.2.65
^Cmach interrupted by signal or user action. Stopping.                  ] 0/28: compiler_builtins(build.rs), core,...

C:\Users\manis\dev\servo>rustup self update
info: checking for self-updates
info: downloading self-update
info: rustup updated successfully to 1.20.2

C:\Users\manis\dev\servo>python mach build --uwp --win-arm64 -r
info: component 'rust-src' is up to date
    Updating crates.io index
warning: Patch `rustc-std-workspace-std v1.99.0 (C:\Users\manis\.rustup\toolchains\nightly-2019-11-16-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\tools\rustc-std-workspace-std)` was not used in the crate graph.
Check that the patched package version and available features are compatible
with the dependency requirements. If the patch has a different version from
what is locked in the Cargo.lock file, run `cargo update` to use the new
version. This may also occur with an optional dependency that is not enabled.
   Compiling compiler_builtins v0.1.21
   Compiling core v0.0.0 (C:\Users\manis\.rustup\toolchains\nightly-2019-11-16-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\libcore)
   Compiling cc v1.0.47
   Compiling libc v0.2.65
   Compiling autocfg v0.1.7
   Compiling std v0.0.0 (C:\Users\manis\.rustup\toolchains\nightly-2019-11-16-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\libstd)
   Compiling hashbrown v0.6.3
   Compiling backtrace-sys v0.1.32
   Compiling unwind v0.0.0 (C:\Users\manis\.rustup\toolchains\nightly-2019-11-16-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\libunwind)
   Compiling rustc-std-workspace-core v1.99.0 (C:\Users\manis\.rustup\toolchains\nightly-2019-11-16-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\tools\rustc-std-workspace-core)
   Compiling cfg-if v0.1.10
   Compiling alloc v0.0.0 (C:\Users\manis\.rustup\toolchains\nightly-2019-11-16-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\liballoc)
   Compiling rustc-demangle v0.1.16
   Compiling panic_abort v0.0.0 (C:\Users\manis\.rustup\toolchains\nightly-2019-11-16-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\libpanic_abort)
   Compiling backtrace v0.3.40
   Compiling rustc-std-workspace-alloc v1.99.0 (C:\Users\manis\.rustup\toolchains\nightly-2019-11-16-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\tools\rustc-std-workspace-alloc)
error[E0635]: unknown feature `compiler_builtins_lib`
   --> C:\Users\manis\.rustup\toolchains\nightly-2019-11-16-x86_64-pc-windows-msvc\lib\rustlib\src\rust\src\libstd\lib.rs:247:12
    |
247 | #![feature(compiler_builtins_lib)]
    |            ^^^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0635`.
error: could not compile `std`.

To learn more, run the command again with --verbose.
error: `"cargo" "build" "--release" "--manifest-path" "C:\\Users\\manis\\AppData\\Local\\Temp\\xargo.knWhSgxFtJnz\\Cargo.toml" "--target" "aarch64-uwp-windows-msvc" "-p" "std"` failed with exit code: Some(101)
note: run with `RUST_BACKTRACE=1` for a backtrace
Build FAILED in 0:00:29

C:\Users\manis\dev\servo>
```

</details>

Seems like https://github.com/servo/servo/pull/24749 and #24750 updated us to a rustc which doesn't build on UWP, and Servo's UWP build builds rustc locally.

I'm unclear why this didn't fail CI, though.

cc @saschanaz @SimonSapin @jdm

--- a/python/servo/packages.py
+++ b/python/servo/packages.py
@@ -11,5 +11,5 @@ WINDOWS_MSVC = {
     "openssl": "111.3.0+1.1.1c-vs2017-2019-09-18",
     "gstreamer-uwp": "1.16.0.5",
     "openxr-loader-uwp": "1.0",
-    "xargo": "v0.3.16",
+    "xargo": "v0.3.17",
 }

